packages/seacas/libraries/ioss/src/Ioss_SideSet.C

```diff
@@ -170,3 +170,49 @@ void Ioss::SideSet::block_membership(std::vector<std::string> &block_members)
   }
   block_members = blockMembership;
 }
+
```

Comment:
Even if both `equal` and `operator==` are needed, we shouldn't be duplicating code between them.

Reply:
Addressed in branch `tkordenbrock-seacas/add-ioshell-compare`.